README.md

```diff
@@ -67,23 +67,39 @@ $ $GOPATH/bin/terraform-provider-ionoscloud
 ...
 ```
 
+## Testing the Provider
+
+⚠️ **Warning:** Acceptance tests create real resources, and often cost money to run.
+
 In order to test the provider, you can simply run `make test`.
 
-```sh
+``` bash
 $ make test
 ```
 
 In order to run the full suite of Acceptance tests, run `make testacc TAGS=all`.
 
-Tests can be split using build tags: all(all tests), compute(all compute-engine tests), dbaas, k8s, server, lan, ipblock.
+``` bash
+$ make testacc TAGS=all
+```
 
-⚠️ **_Note: Acceptance tests create real resources, and often cost money to run._**
+### Test Tags
```

Comment:
From my perspective, there is a **lot** of information about testing. At a first look, it seemed to me we have information about testing for 1/2 of the README.md file. Personally, I would wrap some of the info with a toggle button. 

For me, the main point with testing info in a README is to be _transparent_ about what tests (unit, e2e, acceptance) are present, how to run them locally (with one note about the tags) and CIs.